x-pack/plugins/reporting/common/types/base.ts

```diff
@@ -22,17 +25,26 @@ export type BaseParams = Ensure<
   SerializableRecord
 >;
 
+/**
+ * Report job parameters that an application must return from its
+ * getSharingData function.
+ */
 export type BaseParamsV2 = BaseParams & {
   locatorParams: LocatorParams[];
 };
 
-// base params decorated with encrypted headers that come into runJob functions
+/**
+ * @deprecated
+ */
 export interface BasePayload extends BaseParams {
   headers: string;
   spaceId?: string;
   isDeprecated?: boolean;
 }
 
+/**
+ * Report job parameters, after they are processed in the request handler.
```

Comment:
What do you think about something like:

```suggestion
 * The output of processing a report job.
```

Reply:
I think this suggestion isn't accurate. It sounds too closely like a description of a "completed" job